src/main/java/com/northconcepts/datapipeline/foundations/examples/datamapping/MapDataFromSourceToTargetUsingAutoFieldMapping.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2006-2023 North Concepts Inc.  All rights reserved.
+ * Proprietary and Confidential.  Use is subject to license terms.
+ * 
+ * https://northconcepts.com/data-pipeline/licensing/
+ */
+package com.northconcepts.datapipeline.foundations.examples.datamapping;
+
+import com.northconcepts.datapipeline.core.Record;
+import com.northconcepts.datapipeline.foundations.datamapping.DataMapping;
+import com.northconcepts.datapipeline.foundations.datamapping.DataMappingResult;
+import com.northconcepts.datapipeline.internal.expression.DefaultExpressionContext;
+
+public class MapDataFromSourceToTargetUsingAutoFieldMapping {
```

Comment:
Not bad name, but let's use `MapDataUsingAutoMappingAndExcludedFields`.

Reply:
Done.